kubetest/main.go

```diff
@@ -105,15 +105,20 @@ type options struct {
 	skipRegex          string
 	soak               bool
 	soakDuration       time.Duration
-	sshUser            string
-	stage              stageStrategy
-	test               bool
-	testArgs           string
-	testCmd            string
-	testCmdName        string
-	testCmdArgs        []string
-	up                 bool
-	upgradeArgs        string
+	// sshProxyInstanceName is the name of the vm instance which ip address will be used to set the
+	// KUBE_SSH_BASTION env. If set, it will result in proxying ssh connections in tests through the
+	// "bastion". It's useful for clusters with nodes without public ssh access, e.g. nodes without
+	// public ip addresses.
+	sshProxyInstanceName string
```

Comment:
I would like to see this prefixed with `gcp` because as currently implemented this is gcp-specific; there is no way to use this flag to specify a bastion instance for a number of other deployers

Reply:
eg: `gcpSSHProxyInstanceName`

Reply:
Good point, done.